hrdf/a-2084-01-parallel-gripper.hrdf

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<robot version="1.1.0">
+    
+  <actuator type="X5-4"/>
+  <bracket type="X5HeavyRightOutside"/>
+  <actuator type="X5-9"/>
+  <link type="X5" extension="0.325" twist="pi/2"/>
+  <actuator type="X5-4"/>
+  <link type="X5" extension="0.325" twist="0"/>
+  <actuator type="X5-1"/>
+  <gripper type="X5Parallel"/>
```

Comment:
In the HRDF, this tag will be `end-effector`, not `gripper`.

Reply:
Fixed